File: Freight.API/Libraries/Freight.API.Common/Model/Customer/Customer.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Freight.API.Common.Model.Customer
{
    public class Customer
    {
        public int CustomerID { get; set; }
        public int BusinessID { get; set; }
        public string CompanyName { get; set; }
        public string Email { get; set; }
        public string APEmail { get; set; }
        public int UpdatedBy { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
    }
}

File: Freight.Web/freight/src/app/_shared/model/customer.ts
export class Customer {
  constructor(
    public customerID: number,
    public businessID: number,
    public companyName: string,
    public email: string,
    public apEmail: string,
    public updatedBy: any,
    public createdBy: any,
  ) { }
}
export class Contact {
  constructor(
    public contactID: number,
    public customerID: number,
    public businessID: number,
    public fullName: string,
    public email: string,
    public title: string,
    public updatedBy: any,
    public createdBy: any,
  ) { }
}